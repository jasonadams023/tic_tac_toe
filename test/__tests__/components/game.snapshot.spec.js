import React from 'react';
import Game from '../../../app/components/game';

describe('Game', () => {
    it('displays a new game', () => {
        const wrapper = shallow (
            < Game />
        );

        expect(wrapper).toMatchSnapshot();
    });

    it('displays a cat\'s game', () => {

    });
});