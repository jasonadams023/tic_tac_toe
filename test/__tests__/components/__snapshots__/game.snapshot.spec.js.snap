// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game displays a new game 1`] = `
<div
  className="game"
>
  <div
    className="game-board"
  >
    <Board
      onClick={[Function]}
      squares={
        Array [
          Array [
            null,
            null,
            null,
          ],
          Array [
            null,
            null,
            null,
          ],
          Array [
            null,
            null,
            null,
          ],
        ]
      }
    />
  </div>
  <div
    className="game-info"
  >
    <div>
      Next player: X
    </div>
    <ol>
      <li
        key="0"
      >
        <a
          href="#"
          onClick={[Function]}
        >
           
          Game Start
           
        </a>
      </li>
    </ol>
  </div>
</div>
`;
